Manito/Manito/Screens/Interaction/View/OpenManittoView.swift

```diff
@@ -0,0 +1,151 @@
+        let collectionView = UICollectionView(frame: .zero, collectionViewLayout: self.collectionViewFlowLayout)
+        collectionView.backgroundColor = .clear
+        collectionView.showsVerticalScrollIndicator = false
+        collectionView.isScrollEnabled = false
+        collectionView.register(cell: OpenManittoCollectionViewCell.self,
+                                forCellWithReuseIdentifier: OpenManittoCollectionViewCell.className)
+        return collectionView
+    }()
+    private let titleLabel: UILabel = {
+        let label = UILabel()
+        label.font = .font(.regular, ofSize: 34)
+        label.text = TextLiteral.openManittoViewControllerTitle
+        return label
+    }()
+    private let popupView: OpenManittoPopupView = OpenManittoPopupView()
+
+    // MARK: - property
+
+    private let totalCount = 10.0
+    private(set) var randomIndex = -1 {
+        didSet {
+            self.manittoCollectionView.reloadData()
+        }
+    }
+
+    // MARK: - init
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        self.setupLayout()
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - func
+
+    private func setupLayout() {
+        self.addSubview(self.titleLabel)
+        self.titleLabel.snp.makeConstraints {
+            $0.top.equalTo(self.safeAreaLayoutGuide).inset(57)
+            $0.leading.equalToSuperview().inset(16)
+        }
+
+        self.addSubview(self.manittoCollectionView)
+        self.manittoCollectionView.snp.makeConstraints {
+            $0.top.equalTo(self.titleLabel.snp.bottom)
+            $0.leading.trailing.bottom.equalTo(self.safeAreaLayoutGuide)
+        }
+
+        self.addSubview(self.popupView)
+        self.popupView.snp.makeConstraints {
+            $0.edges.equalToSuperview()
+        }
+    }
+
+    private func animateManittoCollectionView(with friendList: FriendList,
+                                              _ manittoIndex: Int,
+                                              _ manittoNickname: String) {
+        let timeInterval: Double = 0.3
+        let durationTime: Double = timeInterval * self.totalCount
+        let delay: Double = 1.0
+
+        DispatchQueue.main.asyncAfter(deadline: .now() + delay, execute: {
+            UIView.animate(withDuration: durationTime, animations: {
+                self.performRandomShuffleAnimation(with: timeInterval, friendList)
+            }, completion: { _ in
+                let deadline: DispatchTime = .now() + delay + durationTime
+                self.performOpenManittoAnimation(with: deadline, manittoIndex, manittoNickname)
+            })
+        })
+    }
+
+    private func performRandomShuffleAnimation(with timeInterval: TimeInterval, _ friendList: FriendList) {
+        guard let count = friendList.count else { return }
+        var countNumber = 0
```

Comment:
p1;

여기도 타입 명시 부탁드려요!
```suggestion
        var countNumber: Int = 0
```

Reply:
[01b365f](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/433/commits/01b365f67a7f7470ad78c5a1722258463b9cf5eb)

타입 명시했습니다!